src/ctl.c

```diff
@@ -867,6 +868,7 @@ static const ctl_named_node_t experimental_node[] = {
 	{NAME("hooks"),		CHILD(named, experimental_hooks)},
 	{NAME("utilization"),	CHILD(named, experimental_utilization)},
 	{NAME("arenas"),	CHILD(indexed, experimental_arenas)},
+	{NAME("arenas_create_ext"),	CTL(experimental_arenas_create_ext)},
```

Comment:
I've decided to create a new mid-level `arenas_create_ext` leaf CTL because the `arenas` is an indexed node - this is different than the `arenas.`. Looks like an oversight, but I'm not sure - I'm open to suggestions.

Reply:
No, I think that makes sense. Ideally we would have named the existing `experimental.arenas` `experimental.arena` instead, but no need to fix that here (it would be difficult for some of the other experimental functionality experiments).